src/main/webapp/app/exercises/quiz/participate/quiz-participation.component.ts

```diff
@@ -441,10 +441,12 @@ export class QuizParticipationComponent implements OnInit, OnDestroy {
     }
 
     checkForQuizEnd() {
-        const running = this.mode === 'live' && !!this.quizBatch && this.remainingTimeSeconds >= 0 && this.quizExercise?.quizMode !== QuizMode.SYNCHRONIZED;
```

Comment:
Why is the SYNCHRONIZED not relevant anymore?

Reply:
In live mode, we want to save at the end of the quiz, which should go for all quizmodes (batched/synchronized/individual) - the SYNCHRONIZED here was for an old condition that displayed an (in my opinion rather misleading) alert at the end of the quiz.